source/css/_common/components/third-party/related-posts.styl

```diff
@@ -0,0 +1,25 @@
+.popular-posts-header {
+  margin-top: $post-eof-margin-bottom;
+  margin-bottom: 10px;
+  font-size: $font-size-headings-base;
+  border-bottom: 1px solid $gainsboro;
+  display: block;
+}
+
+ul.popular-posts {
+  padding: 0;
+
+  .popular-posts-item {
+    // list-style: none;
+    margin-left: 2em;
+    .popular-posts-title {
+      display: block;
+      h3 {
```

Comment:
H3 here? And if will `seo` false and main then title will H1 and no headers will used — H1-H3 space?

Reply:
@ivan-nginx It is generated by the plugin (https://github.com/theme-next/hexo-theme-next/pull/109/files#diff-0e030eb6dd7b9d25bfcba06bc8ea5609R337), otherwise, do we need to custom the template?

Reply:
H3 in the bottom bad for SEO. For example, if `SEO` will `true` and we will have:
```
H1 (site_subtitle)
  H2 (title)
    H3
      H4
        H5
    H3
      H4
    H3 (related_posts)
```
It's already fine with SEO. But if `SEO` `false`, then we can get fine result:
```
H1 (title)
  H2
    H3 (related_posts)
```
But we also *may* get bad result:
```
H1 (title)
  ???
    H3 (related_posts)
```
So, with `SEO` diabled & `releated_posts` enabled user can totally break his seo.
***
Headers can go forward only seriatim. And can go back with any new position (header reseting). They can't go forward bigger then 1 or more step (e.g. H4 header and afet H6).
![seo](https://cdn.almostover.ru/images/2016-08/fix-h1-to-h2-headings-in-posts-of-hexo-themes.png)
***
https://github.com/theme-next/hexo-theme-next/blob/82ac7afd3fae2f59dd35d2314d192249c1496bff/source/css/_schemes/Gemini/index.styl#L131-L160